AudioPlayerSwift/AudioPlayerSwift iOS/AudioPlayerSwiftTests.swift

```diff
@@ -54,6 +54,34 @@ extension AudioPlayerSwiftTests {
         }
     }
 
+    func testInit_withValidData(){
+        guard let path = soundPath else {
+            XCTFail("Unable to continue, 'soundPath'")
+            return
+        }
+
+        let data = try! Data(contentsOf: URL(fileURLWithPath: path))
+
+        do {
+            _ = try AudioPlayer(data: data)
+            XCTAssertTrue(true)
+        } catch {
+            XCTAssertTrue(false)
+        }
+    }
+
+    func testInvalidData(){
+
+        let data = Data()
+
+        do {
+            _ = try AudioPlayer(data: data)
+            XCTAssertTrue(false)
```

Comment:
Ya, I was just trying to follow the pattern already established in the other tests:

https://github.com/tbaranes/AudioPlayerSwift/blob/master/AudioPlayerSwift/AudioPlayerSwift%20iOS/AudioPlayerSwiftTests.swift#L39-L55

Happy to switch it to `XCTFail`